Packs/FiltersAndTransformers/Scripts/ConcatFormat/README.md

```diff
@@ -1,187 +1,87 @@
 
 ## Inputs
 ---
 
-| **Argument Name** | **Description**                                                                                                                          |
-| --- |------------------------------------------------------------------------------------------------------------------------------------------|
-| key | The key to set. Can be a full path such as "Key.ID". If using append=true can also use a DT selector such as "Data\(val.ID == obj.ID\)". |
-| template | The template text which can include DT expressions such as $\{value\}.                                                                   |
-| append | If false, the context key will be overwritten. If set to true, the script will be appended to the existing context key.                  |
-| stringify | Whether to save the argument as a string. The default value is "noop".                                                                   |
-| force | Whether to force the creation of the context. The default value is "false".                                                              |
-| context | The context data that overwrites the XSOAR context.                                                                                      |
-| variable_markers | The pair of start and end markers to bracket a variable name.                                                                            |
-| keep_symbol_to_null | Set to true to not replace a value if the variable is null, otherwise false.                                                             |
+| **Argument Name** | **Description** |
+| --- | --- |
+| value | The text to be concatenated with prefix &amp;amp; suffix  |
+| prefix | A prefix to concat to the start of the argument |
+| suffix | A prefix to concat to the end of the argument |
+| ctx_data | Context Data: Input . \(single dot\) on \`From previous tasks\` to enable to extract the context data. |
+| ctx_inputs | \`inputs\` context: Input 'inputs' \(no quotation\) on \`From previous tasks\` to enable $\{inputs.\} expression in DT. |
+| ctx_inc | \`demisto\` context: Input 'incident' \(no quotation\) on \`From previous tasks\` to enable $\{incident.\} expression in DT. |
+| variable_markers | The pair of start and end markers to bracket a variable name |
+| keep_symbol_to_null | Set to true not to replace a value if the variable is null, otherwise false. |
 
 ## Outputs
 ---
 There are no outputs for this script.
 
 ## Getting Started
 ---
-The script builds a text from a template text which includes variables such as:
-This is a test message for ${user_name}.
-
-The template will be formatted to `This is a test message for John Doe.` by replacing variable parameters.
-
-By default, a variable name starts with `${` and ends with `}` . You can change the start marker and end marker by specifying the `variable_markers` parameter.
+The transformer concatenates prefix and suffix which supports DT expressions to the string.
```

Comment:
Hi, thanks for the donation, why did you reduced the description of the script ? The old one was more extensive, can you please elaborate or use the old one ?